packaging/rpm/microshift.spec

```diff
@@ -208,12 +208,18 @@ install -d %{buildroot}%{_datadir}/selinux/packages/%{selinuxtype}
 install -m644 packaging/selinux/microshift.pp.bz2 %{buildroot}%{_datadir}/selinux/packages/%{selinuxtype}
 
 # Greenboot scripts
+install -d -m755 %{buildroot}%{_datadir}/microshift/functions
+install -p -m644 packaging/greenboot/functions.sh %{buildroot}%{_datadir}/microshift/functions/greenboot.sh
+
 install -d -m755 %{buildroot}%{_sysconfdir}/greenboot/check/required.d
-install -d -m755 %{buildroot}%{_sysconfdir}/greenboot/red.d
 install -p -m755 packaging/greenboot/microshift-running-check.sh %{buildroot}%{_sysconfdir}/greenboot/check/required.d/40_microshift_running_check.sh
+
+install -d -m755 %{buildroot}%{_sysconfdir}/greenboot/red.d
 install -p -m755 packaging/greenboot/microshift-pre-rollback.sh %{buildroot}%{_sysconfdir}/greenboot/red.d/40_microshift_pre_rollback.sh
-install -d -m755 %{buildroot}%{_datadir}/microshift/functions
-install -p -m644 packaging/greenboot/functions.sh %{buildroot}%{_datadir}/microshift/functions/greenboot.sh
+install -p -m755 packaging/greenboot/red.sh %{buildroot}%{_sysconfdir}/greenboot/red.d/40_microshift_set_health.sh
```

Comment:
I think we should not use generic `red.sh` and `green.sh` names in the source repo. 
When you merge these scripts into one, please use `microshift-set-health.sh` name to be consistent with the other existing scripts. It is doing something very specific, hence the name.